### DISCUSSION TITLE
[Feature request] Steps for CDtuner

### DISCUSSION THREAD

**Discussion by miasik** [wkpark/sd-webui-model-mixer#107] [Ideas]
I'm writing about Add details feature. It has two limiters for the stop steps for normal rendering and highres.fix
I'd like to suggest some improving for it.
Start steps. Adding noise from the first step can be destructive even in amount of 1. Having an option to start adding details from any step would be so useful!
Calculated start/stop steps. When the step set is in between 0-1 then consider it as a calculated number from original rendering steps. Example: Stop step is 0.5 and rendering steps is 36. It means that the real stop step is 36*0.5=18

**Comment by wkpark** [comment]
Is this the right place to suggest it? It might be a suggestion for uddetailer, I guess.

**Reply by miasik** [reply]
Omg, I missed the extension :-)

**Reply by miasik** [reply]
It should be there, I'm sorry :-(
https://github.com/hako-mikan/sd-webui-cd-tuner/issues/19
